pkg/scripts/build.sh

```
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+COMPONENT_NAME=$1
```

Shellcheck (0.11.0)

[warning] COMPONENT_NAME appears unused. Verify use (or export if used externally).

(SC2034)